### ISSUE TITLE
Update templates for Rails 6

### ISSUE THREAD

**Issue by vdegove** [lewagon/rails-templates#82]
A few things not relevant anymore to Rails 6 (list to be completed)
* In `app/controllers/application_controller.rb` no need anymore of  `protect_from_forgery with: :exception` as the parameter is moved to ActionController::Base and set to true by default see https://github.com/rails/rails/commit/ec4a836919c021c0a5cf9ebeebb4db5e02104a55
* In `app/views/layouts/application.html.erb` no need of `<%= javascript_include_tag 'application' %>` as all native javascript components of Rails are moved to Webpack

**Comment by hidehiro98** [comment]
I got this error with Rails 6 as well
```
run    rm app/assets/javascripts/application.js from "."
rm: app/assets/javascripts/application.js: No such file or directory
```

**Comment by hidehiro98** [comment]
@vdegove 
First one looks solved by PR #90 and #93 .
